build.sbt

```diff
@@ -164,9 +185,9 @@ lazy val noPublishSettings = Seq(
 
 val algebirdVersion = "0.12.0"
 val bijectionVersion = "0.9.1"
-val utilVersion = "6.34.0"
+def utilVersion(scalaVersionValue: String) = if (scalaVersionValue.startsWith("2.10.")) "6.34.0" else "6.42.0"
```

Comment:
Not sure how important it is to support 2.10. Having two versions of external dependencies makes maintenance harder.